x/params/subspace/subspace_test.go

```diff
@@ -0,0 +1,203 @@
+package subspace_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params/subspace"
+	"github.com/stretchr/testify/suite"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	dbm "github.com/tendermint/tm-db"
+)
+
+type SubspaceTestSuite struct {
+	suite.Suite
+
+	cdc *codec.Codec
+	ctx sdk.Context
+	ss  subspace.Subspace
+}
+
+func (suite *SubspaceTestSuite) SetupTest() {
```

Comment:
the struct conflicts with the `suite` package name:

```suggestion
func (sts *SubspaceTestSuite) SetupTest() {
```

Reply:
Thanks, but I don't want to change this as it's based on the API docs and pretty standard across the rest of the suite testing we have.